File: linalg/vectors/challenge_1.m
A = randn(4, 6);
B = randn(4, 6);

dps = zeros(size(A, 2), 1);
for i = 1: size(A, 2)
    dps(i) = dot(A(:, i), B(:, i));
    disp(dps(i));
end

File: linalg/vectors/challenge_1.py
import numpy as np

m = 4
n = 6

A = np.random.randn(m, n)
B = np.random.randn(m, n)

print(A)
print(B)

dps = np.zeros(n)
for i in range(n):
    dps[i] = np.dot(A[:, i], B[:, i])

print(dps)
